Cargo.toml

```
@@ -1,26 +1,37 @@
 [package]
-name = "meilisearch-tokenizer"
-version = "0.2.9"
+name = "charabia"
+version = "0.3.0"
 license = "MIT"
 authors = ["Many <many@meilisearch.com>"]
-edition = "2018"
+edition = "2021"
+description = "Library used by Meilisearch to tokenize queries and documents"
```

Do we want to specify that it is **the** library used by Meilisearch?
```suggestion
description = "A simple library to detect the language, tokenize the text and normalize the tokens"
```